azure-pipelines-external.yml

```diff
@@ -0,0 +1,150 @@
+pr:
+  branches:
+    include:
+    - '*'  # must quote since "*" is a YAML reserved character; we want a string
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  - group: sonar-dotnet-variables
+  - group: sonarsource-build-variables
+
+stages:
+- stage: buildAndTests
+  displayName: Build and run tests
+  jobs:
+    - job: runUnitTestsJob
```

Comment:
This will run only .Net build, unit tests and integration tests. I don't see a reason for someone from community to change the plugin but we need to make this visible for the hole team that we won't accept changes for the java code or at least the changes are not validated.

Reply:
the idea is to run the full job as well before merging (w/o running the full job, we won't be able to merge anyway as Java ITs are required)

Reply:
Yes, that's true.